### ISSUE TITLE
Uploading from ios hangs

### ISSUE THREAD

**Issue by dottodot** [balderdashy/skipper#42]
My site uses dropzone.js to upload files which is working fine with skipper. But when I try to upload form my iphone it hangs for a while and then I get an error of

```
error: READ error on file image.jpg :: Error: Request aborted
    at IncomingMessage.onReqAborted (/host/node_modules/sails/node_modules/skipper/node_modules/multiparty/index.js:175:17)
    at IncomingMessage.EventEmitter.emit (events.js:92:17)
    at abortIncoming (http.js:1911:11)
    at Socket.serverSocketCloseListener (http.js:1923:5)
    at Socket.EventEmitter.emit (events.js:117:20)
    at TCP.close (net.js:465:12) [Error: Request aborted]
```

any suggestion on what the cause of this would be?

**Comment by treyrich2001** [comment]
+1. I'm getting this exact same issue, and it's causing a lot of grief. I haven't gotten a chance to dig very deep into it yet, but I do know my client who's being affected by this is extremely annoyed so I'll probably have to dig into it pretty soon. Would love to hear from any core contributors (or @dottodot) on insights into work arounds on this before I duplicate work that's already been done to mitigate the issue. Anyone have any thoughts?

**Comment by dottodot** [comment]
From what I remember the issue was caused by the first release of ios8 and was fixed in one of the updates.

**Comment by treyrich2001** [comment]
Awesome. I'm actually remembering having this issue from before now that you mention it being an iOS bug. That helps. Thanks.

**Comment by sailsbot** [comment]
Thanks for posting, @dottodot.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
